app/views/shared/_header.html.erb

```diff
@@ -7,7 +7,7 @@
       <% end %>
       <nav id="menu" class="col-sm" role="menu">
         <%= link_to "About","#{root_url}about", :class => "btn btn-primary about", :role => "menuitem" %>
-        <%= link_to "How to Submit", "#{root_url}how-to-submit", :class => "btn btn-primary how-to-submit", :role => "menuitem" %>
+        <%= link_to "How to Submit", "#how-to-submit", :class => "btn btn-primary how-to-submit", :role => "menuitem" %>
```

Comment:
I think we want the `#{root_url}` to be part of the URL. Without it the link will work as long as the user is on the home page (e.g. `/describe`) but it will fail if the user tries to select "How to submit" while they are on another page (e.g. on their dashboard `/describe/hc8719`)